admin.go

```diff
@@ -479,7 +503,19 @@ func (ca *clusterAdmin) AlterConfig(resourceType ConfigResourceType, name string
 		ValidateOnly: validateOnly,
 	}
 
-	b, err := ca.Controller()
+	var (
+		b   *Broker
+		err error
+	)
+
+	// AlterConfig of broker must be sent to the broker in question
+	if resourceType == BrokerResource {
```

Comment:
If the resource is  `BrokerResource` but it has no name (ie. describing its default), then the request can be sent to any broker too. Same for `BrokerLoggerResource`.

Reply:
👍